docs/wearables/connecting-providers/auth_types.mdx

```diff
@@ -0,0 +1,264 @@
+---
+title: "Provider Types"
+---
+
+Providers have different authentication flows. Some require a username and password, others require an email address and password, and some require an OAuth flow.
+
+Vital supports the following provider types:
+
+- `OAuth`
+- `Email`
+- `Email + Password`
+
+The majority of providers are `OAuth` providers. These providers require a user to be redirected to a third party website to authenticate. Once authenticated, the user is redirected back to the link widget once connected.
+The `Email` and `Email + Password` providers require a user to enter their email address and password into the link widget. This is then sent to Vital and used to connect to the provider.
```

Comment:
This feels repeated, from line 9, 10 and 11. Perhaps add an explanation of OAuth there alongside Email and password?